App/Sagas/ImageSharingTriggers.ts

```diff
@@ -0,0 +1,56 @@
+import { call, put, select } from 'redux-saga/effects'
+import { ActionType } from 'typesafe-actions'
+import RNFS from 'react-native-fs'
+
+import ProcessingImagesActions, { ProcessingImage, ProcessingImagesSelectors } from '../Redux/ProcessingImagesRedux'
+import UIActions from '../Redux/UIRedux'
+import { insertImage, addToIpfs, uploadArchive, shareWalletImage, addToWallet, shareToThread } from './ImageSharingSagas'
+
+export function * handleSharePhotoRequest(action: ActionType<typeof UIActions.sharePhotoRequest>) {
```

Comment:
This one handled the UI action of sharing a photo, either from the wallet or camera roll.